pages/event-types/index.tsx

```diff
@@ -153,7 +153,11 @@ const EventTypeList = ({ readOnly, types, profile }: EventTypeListProps): JSX.El
                         truncateAfter={4}
                         items={type.users.map((organizer) => ({
                           alt: organizer.name || "",
-                          image: organizer.avatar || "",
+                          image:
+                            (process.env.NEXT_PUBLIC_APP_URL || process.env.BASE_URL) +
+                            "/" +
+                            organizer.username +
```

Comment:
We have a type error here:

> Property 'username' does not exist on type '{ name: string | null; id: number; avatar: string | null; }'.ts(2339)

Reply:
Perhaps the instance didn't get updated but organizer does have the property username, as it is pulled from here:
https://github.com/calcom/cal.com/pull/1759/files#diff-758cb4a85756482b534d35a7806ed3bcdbd2775698e803a83a40773ba1c24582R134